src/pgduckdb_duckdb.cpp

```diff
@@ -25,6 +25,14 @@ namespace pgduckdb {
 DuckDBManager::DuckDBManager() {
 }
 
+#define SET_DUCKDB_OPTION(ddb_option_name)                                                                             \
+	config.options.ddb_option_name = duckdb_##ddb_option_name;                                                         \
+	{                                                                                                                  \
+		std::ostringstream oss;                                                                                        \
+		oss << "[PGDuckDB] Set DuckDB option: '" << #ddb_option_name << "'=" << duckdb_##ddb_option_name;              \
+		elog(DEBUG2, "%s", oss.str().c_str());                                                                           \
```

Comment:
For DEBUG logs it's best to put everything in arguments of the `elog` call, because those are not executed when the log level is lower. So something like this:
```suggestion
		elog(DEBUG2, "[PGDuckDB] Set DuckDB option: '%s'=%s", #ddb_option_name, std::string(oss.str()).c_str());                                                                           \
```

Reply:
Yeah good point - initally I didn't care too much cause I did use `INFO` level, but tests reminded me it wasn't a good idea :-) 
I've reworked it.